ffmpeg/README.md

```diff
@@ -0,0 +1,20 @@
+# Build ffmpeg as static PIE
+
+Script to build ffmpeg as a static PIE binary for use with Unikraft.
```

Comment:
This should not mention `Unikraft`, you are building the `ffmpeg` application as a statically linked ELF to run on Linux.
See the example [here](https://github.com/unikraft/static-pie-apps/tree/master/nginx#build-nginx-as-static-pie).